core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -76,6 +79,28 @@
    * <code>avro.compatibility.level</code>
    */
   public static final String COMPATIBILITY_CONFIG = "avro.compatibility.level";
+  public static final String KAFKASTORE_SECURITY_PROTOCOL_CONFIG =
+      "kafkastore.security.protocol";
+  public static final String KAFKASTORE_SSL_TRUSTSTORE_LOCATION_CONFIG =
+      "kafkastore.ssl.truststore.location";
+  public static final String KAFKASTORE_SSL_TRUSTSTORE_PASSWORD_CONFIG =
+      "kafkastore.ssl.truststore.password";
+  public static final String KAFKASTORE_SSL_KEYSTORE_LOCATION_CONFIG =
+      "kafkastore.ssl.keystore.location";
+  public static final String KAFKASTORE_SSL_KEYSTORE_PASSWORD_CONFIG =
+      "kafkastore.ssl.keystore.password";
+  public static final String KAFKASTORE_SSL_KEY_PASSWORD_CONFIG =
+      "kafkastore.ssl.key.password";
+  public static final String KAFKASTORE_SSL_ENABLED_PROTOCOL_CONFIG =
+      "kafkastore.ssl.enabled.protocols";
+  public static final String KAFKASTORE_SSL_KEYSTORE_TYPE_CONFIG =
+      "kafkastore.ssl.keystore.type";
+  public static final String KAFKASTORE_SSL_PROTOCOL_CONFIG =
+      "kafkastore.ssl.protocol";
+  public static final String KAFKASTORE_SSL_PROVIDER_CONFIG =
+      "kafkastore.ssl.provider";
+  public static final String KAFKASTORE_SSL_TRUSTSTORE_TYPE_CONFIG =
```

Comment:
Nitpick: it would be a bit nicer if all related configs were together (eg all truststore configs were grouped, all keystore configs were grouped, etc.)